Ownership/AC - South/Worthing.txt

```diff
@@ -19,8 +19,8 @@ GUEST:LD:*:EGHH
 
 SECTOR:London AC Worthing 2 (215-660):21500:66000
 OWNER:LS:LWS:LSC:L
-ALTOWNER:Observing LON_S_CTR:LGS:O:LM:LS:LWS:LSC:L
-ALTOWNER:Observing London FIR:LGS:O:LM:LS:LWS:LSC:L
+ALTOWNER:Observing LON_S_CTR:LGS:O:LMEN:LS:LWS:LSC:L
```

Comment:
Could you add the Observing LON_SC_CTR ownership to all of these sub-sectors please?